src/SCRIPTS/BF/HORUS/rates2.lua

```diff
@@ -5,7 +5,7 @@ return {
    title          = "Rates (2/2)",
    reboot         = false,
    eepromWrite    = true,
-   minBytes       = 23,
```

Comment:
This will break backwards compatibility with older versions of the firmware. The minimum length for supported versions of the firmware is still 23 bytes.